superset-frontend/spec/javascripts/messageToasts/components/Toast_spec.jsx

```diff
@@ -31,32 +30,27 @@ const props = {
 const setup = overrideProps => mount(<Toast {...props} {...overrideProps} />);
 
 describe('Toast', () => {
-  it('should render an Alert', () => {
+  it('should render', () => {
     const wrapper = setup();
-    expect(wrapper.find(Alert)).toExist();
+    expect(wrapper.find('div')).toExist();
```

Comment:
It might be good to make these a little more specific, like perhaps a `data-test` attribute